paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/AutoRefreshSettingsDefinition.kt

```diff
@@ -0,0 +1,7 @@
+package com.stripe.android.paymentsheet.example.playground.settings
+
+internal object AutoRefreshSettingsDefinition : BooleanSettingsDefinition(
```

Comment:
Seems like this should be saveable also. I'd like to not have to turn it off every time I enter the playground.

Reply:
Is `BooleanSettingsDefinition` not saveable by default?

Reply:
It is, my bad!